gplog/gplog.go

```diff
@@ -390,7 +391,12 @@ func Fatal(err error, s string, v ...interface{}) {
 	message := ""
 	stackTraceStr := ""
 	if err != nil {
-		message += fmt.Sprintf("%v", err)
+		if _, ok := err.(*pgconn.PgError); !ok {
```

Comment:
We should print the HINT too. Also, ERRCODE too (if not already present in the message etc)

Reply:
I think errcode will already be included somewhere

Reply:
This Error function gives you Severity, Message, and Error Code https://github.com/jackc/pgconn/blob/v1.14.3/errors.go#L51

Reply:
gpbackup is doing this around where we get the COPY errors if that's helpful for reference

```
		// The COPY ON SEGMENT error might contain useful CONTEXT output
		if pgErr, ok := err.(*pgconn.PgError); ok && pgErr.Where != "" {
			errStr = fmt.Sprintf("%s: %s", errStr, pgErr.Where)
		}

		err = errors.Wrap(err, errStr)
```